src/main/clojure/jarl/exceptions.clj

```diff
@@ -19,3 +20,36 @@
 
 (defn type-ex [message & args]
   (TypeException. (apply format message args)))
+
+; Must be a better way to do this
```

Comment:
If we prefix with `FIXME:` it'll be easier to find and fix stuff like this later.

Reply:
Yep, I'll add that to the next PR.